source_control/subversion.py

```diff
@@ -115,8 +122,20 @@ def _exec(self, args):
         if self.password:
             bits.extend(["--password", self.password])
         bits.extend(args)
-        rc, out, err = self.module.run_command(bits, check_rc=True)
-        return out.splitlines()
+        if check_rc:
+            rc, out, err = self.module.run_command(bits, check_rc)
+            return out.splitlines()
+        else:
+            rc, out, err = self.module.run_command(bits, check_rc)
+            return rc
```

Comment:
To make this a little more DRY, move the line `rc, out, err = self.module.run_command(bits, check_rc)` in front of the if/else block (so that there is only one instance of this command).